tools/bcr_validation.py

```diff
@@ -307,6 +313,14 @@ def main(argv=None):
     "--fix",
     action="store_true",
     help="Should the script try to fix the detected validation errors.")
+  parser.add_argument(
+    "--skip_validation",
```

Comment:
how is a user meant to pass this flag? I suppose you probably have some other work in-flight to wire this up to GH PR labels or something?

Reply:
> I'll soon integrate this with https://github.com/bazelbuild/continuous-integration/blob/master/buildkite/bazel-central-registry/bcr_presubmit.py with github labels

Basically, if necessary, a BCR maintainer can set github labels for PRs to bypass those checks. Working on it.